staging/src/k8s.io/client-go/pkg/api/types.go

```diff
@@ -584,10 +584,28 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// If nothing exists at the given path, an empty directory will be created there. Otherwise, behaves like HostPathExists
```

Comment:
Since these comments are describing the methods on these consts, should the comments be moved to the err lines?

The bulk of these comments are repeated and might not be necessary when reading through the function calls.